frontend/packages/shared/src/contexts/ServicesContext.tsx

```diff
@@ -7,11 +7,9 @@ import type * as mutations from '../api/mutations';
 import { ReactQueryDevtools } from '@tanstack/react-query-devtools';
 import type { ToastOptions } from 'react-toastify';
 import { ToastContainer, Slide, toast } from 'react-toastify';
-import { ErrorBoundary } from 'react-error-boundary';
```

Comment:
I could not delete this dependency from the repository, since it's used on the dashboard. 😊